Code/Editor/Plugins/ProjectSettingsTool/ProjectSettingsToolWindow.cpp

```diff
@@ -441,20 +452,37 @@ namespace ProjectSettingsTool
                 ));
             break;
         case PlatformId::Android:
-            m_platformPropertyEditors[platIdValue]->EnumerateInstances(AZStd::bind
+        {
+            auto* androidSettings = m_settingsContainer->GetPlatformData(plat);
+            if (androidSettings == nullptr ||
```

Comment:
suggestion: To be consistent with the line further down (`dict`) should we make this `!androidSettings`?

Reply:
Sure